plugins/github.py

```diff
@@ -3,13 +3,16 @@
 import os
 
 
-list_of_pull_requests=[]
+list_of_pull_requests = []
+
+
+def get_pull_requests(username, repository):
 
-def get_pull_requests(username,repository):
     github_url = "https://api.github.com"
-    r = requests.get(os.path.join(github_url, "repos", username, repository, "pulls"))
+    r = requests.get(os.path.join(github_url, "repos", repository, "pulls"))
     js = json.loads(r.content)
     for i in range(0, len(js)):
```

Comment:
Here `js` is a list. We can iterate over items in a list without `range()`. Then this `for` loop would be:
```
    for pull in js:
        if username == pull['user']['login']:
            if 'title' in pull.keys():
                list_of_pull_requests.append(pull['title'])
```